Assets/HoloToolkit/UX/Scripts/Buttons/AnimControllerButton.cs

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+using System;
+using System.Collections;
```

Comment:
we don't need `System.Collections`

Reply:
Hehe, good call.  I used to always just add system and system.collections to everything just in case :).  Big clean up commit coming.

thanks,
-pat